### PR TITLE
Initial address creation docs

### PR DESCRIPTION
**Pull request by GabrielBlondeau-Kirby**
Initial address creation docs.

-------

`raw` attribute has no description for a moment.
`service_address_id` same

### CODE REVIEW

**Comment by GabrielBlondeau-Kirby** [comment]
Need review :)

File: source/includes/_address.md [hunk 1]
```diff
@@ -1,3 +1,158 @@
...
+
+Attribute | Type | Description
+--------- | ---- | -----
+street_name | `string` | The street name of Address
+building | `string` | The building of Address
+zip_code | `string` | The zip code of Address
+city | `string` | The city of Address 
+country | `string` | The country of Address 
+geo | `GeoPoint` | The geo position of Address
+raw | `dynamic` | ##IDK how to describe this
```
Comment (KristopherJordan) [inline source/includes/_address.md]: The field `raw` is the raw google map output from a query.

File: source/includes/_address.md [hunk 2]
```diff
@@ -1,3 +1,158 @@
...
+alias | `string` | The alias of Address
+service  | `string` | The service used for Address. _default google_
+service_address_id  | `string` | 
+
+### service
+
+Possibilities |
+-------------
+google
+pca
```
Comment (KristopherJordan) [inline source/includes/_address.md]: Don't think we should give the users the opportunity to chose between google or pca. By default we use google

File: source/includes/_address.md [hunk 3]
```diff
@@ -1,3 +1,158 @@
...
 ## Address Object
+
+Attribute | Type | Description
+--------- | ---- | -----
+street_name | `string` | The street name of Address
+building | `string` | The building of Address
+zip_code | `string` | The zip code of Address
+city | `string` | The city of Address 
+country | `string` | The country of Address 
+geo | `GeoPoint` | The geo position of Address
```
Comment (pedropregueiro) [inline source/includes/_address.md]: this geo point doesn't mean anything to the general public, it's a python mongo thing. we need to use global types of the API output, which in this case it's JSON. so I think this is a list with longitude and latitude

File: source/includes/_address.md (hunk 1 above)
Comment (pedropregueiro) [inline source/includes/_address.md]: it's not `dynamic`, it's an `object`

File: source/includes/_address.md [hunk 4]
```diff
@@ -1,3 +1,158 @@
...
+raw | `dynamic` | ##IDK how to describe this
+alias | `string` | The alias of Address
+service  | `string` | The service used for Address. _default google_
+service_address_id  | `string` |
```
Comment (pedropregueiro) [inline source/includes/_address.md]: the identifier on the provider side (google id or pca id)

**Pull request merged**